masterlist.yaml

```diff
@@ -3637,11 +3637,18 @@ plugins:
     inc:
     - 'jsawyer.esp'
     - 'Project Nevada - Rebalance.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Literally Who''s Patch Emporium'
+          - '[ sawyerbatty - FafnirUniqueTTWOverhaul ](https://www.nexusmods.com/newvegas/mods/83169/)'
```

Comment:
Use `[sawyerbatty - FafnirUniqueTTWOverhaul](https://www.nexusmods.com/newvegas/mods/83169/)` instead (empty spaces before and after removed).